create-cookie-keys.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
```

Comment:
Checking a fixed key into source control is not great for a customer example. Also this enables testing with old keys in the browser, to demonstrate reliability.